main.py

```diff
@@ -572,7 +664,7 @@ def process_config(config):
     if not config.workdir and config.machine == "cluster":
         config.workdir = "/home/kilgourm/scratch/learnerruns"
     elif not config.workdir and config.machine == "local":
-        config.workdir = "/home/mkilgour/learnerruns"  # "C:/Users\mikem\Desktop/activeLearningRuns"  #
+        config.workdir = "C:\mila\learnerruns"  # "C:/Users\mikem\Desktop/activeLearningRuns"  # "/home/mkilgour/learnerruns"  #depending on the user
```

Comment:
that a path is hardcoded isn't nice, but we should not keep updating this ^^

use --workdir <yourworkdir>